### PR TITLE
Update dependencies to allow Symfony 4.x-components.

### PR DESCRIPTION
**Pull request by dbrumann**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -18,20 +18,20 @@
...
+        "symfony/class-loader": "~2.1||~3.0||~4.0",
         "psr/container": "^1.0",
         "container-interop/container-interop": "^1.2"
     },
 
     "require-dev": {
-        "symfony/process": "~2.5|~3.0",
-        "phpunit/phpunit": "^4.8.36|^6.3",
+        "symfony/process": "~2.5|~3.0|~4.0",
+        "phpunit/phpunit": "^4.8.36|^6.3"
```
Comment (thecrimpmaster) [inline composer.json]: you're missing a comma here

Comment (ciaranmcnulty) [reply composer.json]: I was so pleased about resolving the conflict via github, too

**Comment by ciaranmcnulty** [comment]
I'm not sure why Scrutinizer is complaining here - looks like the issues were all pre-existing

**Comment by everzet** [comment]
Thank you very much for contribution and sorry for a wait!

**Comment by Nyholm** [comment]
Thank you for merging

**Pull request merged**